SimTracker/TrackAssociatorProducers/python/quickTrackAssociatorByHits_cfi.py

```diff
@@ -20,3 +20,6 @@
     quickTrackAssociatorByHits.associateStrip = False
     quickTrackAssociatorByHits.associatePixel = False
     quickTrackAssociatorByHits.useClusterTPAssociation = False
+
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, pixelSimLinkSrc = cms.InputTag("simSiPixelDigis","Pixel") )
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, stripSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker") )
```

Comment:
I'm fine with either way. Probably at some point the `quickTrackAssociatorByHits` parameters need to be changed though (IIRC the main user of the parameters is FastSim).